src/main/native/jgskit.mak

```diff
@@ -162,46 +138,61 @@ TARGET = ${HOSTOUT}/libjgskit.so
 GSK8ICCS64=jgsk8iccs_64
 GSK8ICCS=jgsk8iccs
 
-all: headers ${TARGET}
+all : ${TARGET}
```

Comment:
Any reason why we dont need to build with headers here?

Reply:
If you check further down, you'll see that `${TARGET}` depends on `${OBJS}` and, finally, `${OBJS}` depends on `headers`. It was changed to ensure that the makefile is thread-safe, as order of requirements doesn't guarantee precedence when running using multiple threads.

Reply:
make sense